proto/vectorindex.proto

```diff
@@ -66,6 +66,81 @@ message _MetadataRequest {
     }
 }
 
+message _AndExpression {
+    _FilterExpression first_expression = 1;
+    _FilterExpression second_expression = 2;
+}
+
+message _OrExpression {
+    _FilterExpression first_expression = 1;
+    _FilterExpression second_expression = 2;
+}
+
+message _NotExpression {
+    _FilterExpression not_expression = 1;
+}
+
+message _EqualsExpression {
+    string field = 1;
+    oneof value {
+        string string_val = 2;
```

Comment:
style nit: to be consistent with the rest of the file, could you rename these to `*_value`? (see metadata above)

Reply:
modified! https://github.com/momentohq/client-protos/pull/246/commits/0314a5fb4975ca699ac9f6fd2c8194e7d343375a